api/config/config.go

```diff
@@ -1,6 +1,21 @@
 package config
 
-// ConfigValues provide default config values
+const (
+	defaultStartGRPCServer = false
+	defaultGRPCServerPort  = 9091
+	defaultStartJSONServer = false
+	defaultJSONServerPort  = 9090
+)
+
+// Config defines the api config params
+type Config struct {
+	StartGrpcServer bool `mapstructure:"grpc-server"`
+	GrpcServerPort  int  `mapstructure:"grpc-port"`
+	StartJSONServer bool `mapstructure:"json-server"`
+	JSONServerPort  int  `mapstructure:"json-port"`
+}
+
+// ConfigValues set default values
 var ConfigValues = Config{
```

Comment:
Do we really need this now that we have DefaultConfig()?

Reply:
We don't but it is being used by the package like hardcoded, so removing would mean having to refactor the package. And currently I am trying to keep the PR small. I can create a separate PR to refactor and fix.